charts/opensearch/README.md

```diff
@@ -66,6 +66,7 @@ helm uninstall my-release
 | `minimumMasterNodes`               | The value for `discovery.zen.minimum_master_nodes`                                                                                 | `2`                                             |
 | `nameOverride`                     | Overrides the `clusterName` when used in the naming of resources                                                                                                                                                                                          | `""`                                            |
 | `networkHost`                      | Value for the `network.host OpenSearch setting`                                                                                                                                                                                                      | `0.0.0.0`                                       |
+| `networkPolicy.create`                      | Enable network policy creation for OpenSearch | `false`
```

Comment:
You have additional spaces leading to `| Enable......` as well.

Reply:
on this one, i have a question, the ideia is  to ident the '|' after the `enable` like is on the line above?

Reply:
I think your new application is good. Thanks.